Lesson2

kurmangaliyev-a opened the pull request:
Какие-то вещи сделал на своё усмотрение (наведение на крестик, фотки, и тп), поэтому не уверен, что всё красиво. Всё что на макете сверстал.

kurmangaliyev-a committed:
normalize.css

kurmangaliyev-a committed:
photosBlock

Добавил поведение наведения для фото (в фигме не прописано, просто для себя, если не нужно, удалю)

kurmangaliyev-a committed:
Set theme jekyll-theme-slate

kurmangaliyev-a committed:
closeModal button

kurmangaliyev-a committed:
Merge branch 'lesson2' of https://github.com/kurmangaliyev-a/Kolesa-Upgrade-FE-2021 into lesson2

kurmangaliyev-a committed:
Input radibuttons added

Добавил радио-кнопки, цвета, кнопку заказать, и балы

kurmangaliyev-a committed:
pixel Perfect

Добавил фоновую картинку
Размеры,
описания деталей и как выбрать размер

kurmangaliyev-a committed:
background Your Balance

На старом мониторе не увидел фон на табличке. Здесь добаил

airtwister commented:
Всегда должен быть синий бэкграунд у выбранного элемента)

![image](https://user-images.githubusercontent.com/5547378/134868749-39723396-1db7-49d9-bd1a-dc1891e7d41d.png)

airtwister commented:
нет бордер-радиуса

![image](https://user-images.githubusercontent.com/5547378/134869083-b3bc1dd7-2175-454b-9935-2aa787c430f5.png)

airtwister commented:
по макету серая обводка 

![image](https://user-images.githubusercontent.com/5547378/134869143-fefb7308-177c-4f29-ba8a-876e236aa114.png)

kurmangaliyev-a committed:
Пофикшены мелкие баги

Изменён фон colorList (всегда голубой)
Рамка для кнопок с размером одежды
Бордер радус таблицы "Твой баланс"

kurmangaliyev-a commented:
Спасибо, исправил

kurmangaliyev-a commented:
Спасибо, исправил

kurmangaliyev-a committed:
Поправил отступы во вкладке "Твой баланс:"

Поправил отступы во вкладке "твой баланс"
